pages/sdk-onramp-kit/guides/monerium.md

```diff
@@ -74,17 +75,15 @@ After that, the Monerium UI, will wait until it detects the Safe address has sig
 
 Note. If you use the `sandbox` environment, you can test this flow without KYC in `Goerli`. To use the mainnets for `ethereum`, `gnosis` or `polygon` in production, you need to create an account in the Monerium dashboard and complete KYC. The `production` environment involves real money, whereas the `sandbox` environment uses fake money and test tokens.
 
-3. Once you authenticate through the Monerium UI, a window will appear asking for permission to access your information. If you accept, control will be returned to the `redirectUrl` you specified in the `open` method.
+3. Once you authenticate through the Monerium UI, a window will appear asking for permission to access your information. If you accept, control will be returned to the `redirectUrl` you specified in the `MoneriumPack` constructor.
 
-4. The `redirectUrl` used in the `open` will be reached again with an extra query string parameter `code`. This is the authorization code you need to exchange in order to gain access to your Monerium account.
-
-Re-initialize the `MoneriumPack` using the `init` method and exchange the code, as per Step 1.
+4. Once we reach the `redirectUrl` we can call open again, now without using arguments.
 
 ```typescript
-const safeMoneriumClient = await moneriumPack.open({ authCode: '<The querystring code parameter>' })
+const safeMoneriumClient = await moneriumPack.open()
```

Comment:
If `safeApiKit` was replaced with `apiKit`, `safeMoneriumClient` should be replaced with `moneriumClient`.

However, I'm not a big fan of doing this, as `safe` adds context for all the variable names.

In the API Kit case, for example, `safeApiKit` is more descriptive, as multiple APIs can be consumed from the same file. `apiKit` is too generic, but this one is from **Safe**. Same with `safeProtocolKit`, `safeMoneriumClient`, etc.

Reply:
in the context of the documents for me the `safe` context is implicit. When dealing with external dApps is the third party developer who should choose the proper naming. For me the name is ok matching the package name. In the case of monerium it's ok for me to keep the `safe` prefix as in their SDK  there is  already a `moneriumClient`